.travis.yml

```diff
@@ -17,7 +17,7 @@ python:
     - 2.7
 
 addons:
-    postgresql: "9.4"
+    postgresql: "9.2"
```

Comment:
I guess you can set it to 9.3 now, right? At least if you're trying to match what we use in Socorro.